### PR TITLE
Syncing with main

### PR DESCRIPTION
**Pull request by snova-guhae**
Need to sync with main

### CODE REVIEW

File: lmms_eval/models/llava.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def __init__(
         device_map="cuda:0",
         conv_template="vicuna_v1",
         use_cache=True,
+        tie_weights: bool = True,
```
Comment (snova-jonathanl) [inline lmms_eval/models/llava.py:61]: Should this be true for llava? At least for v1.5 (and llama 2 models in general), this shouldn't be true: https://huggingface.co/llava-hf/llava-1.5-13b-hf/commit/d23149f1544974682263229a9b373e6e0395826d

File: lmms_eval/models/llava_vid.py [hunk 2]
```diff
@@ -59,6 +59,8 @@ def __init__(
         mm_spatial_pool_mode: str = "average",
         overwrite: bool = True,
         video_decode_backend: str = "pyav",
+        delay_load: bool = False,
+        tie_weights: bool = True,
```
Comment (snova-jonathanl) [inline lmms_eval/models/llava_vid.py:63]: Same as above, should this default to True?

**Review by snova-jonathanl — COMMENTED** [review]
Aside from the one comment for llava, everything else lgtm!

File: lmms_eval/models/llava.py (hunk 1 above)
Comment (EtashGuha) [reply lmms_eval/models/llava.py:61]: hm, I think we should merge in and then just make sure to set tie_weights to false if we want to?

File: lmms_eval/models/llava_vid.py (hunk 2 above)
Comment (EtashGuha) [reply lmms_eval/models/llava_vid.py:63]: Same thing as above. We aren't doing video anyways, so let's keep the change.

**Review by EtashGuha — APPROVED** [review]

**Pull request merged**